runtime/src/bank.rs

```diff
@@ -5131,6 +5197,76 @@ pub(crate) mod tests {
         bank
     }
 
+    #[test]
+    fn test_store_account_and_update_capitalization_missing() {
```

Comment:
classical textbookish non-overlapping exhaustive set of unit tests for `store_account_and_update_capitalization`, which is newly promoted from `cfg(test)`.

Reply:
this comment is reposted at https://github.com/solana-labs/solana/pull/14008/files#r538097839